### PR TITLE
pfsense_cert task

### PR DESCRIPTION
**Pull request by cmarodrigues**
Hi,

This PR add the pfsense_cert task to the pfsense ansible module.

This task allows us to create a new certificate signed by internal CA and/or import and existing certificate.

The example to generate a new certificate:

```
- name: Generate new internal certificate
  pfsense_cert:
    method: "internal"
    name: "test"
    ca: "internal-ca"
    keytype: "RSA"
    keylen: 2048
    lifetime: 3650
    dn_country: "PT"
    dn_organization: "Dummy"
    certtype: "user"
    state: present
```
Or to import a existing certificate:

```
- name: Import certificate
  pfsense_cert:
    method: "import"
    name: "test"
    certificate: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0t...
    key: |
      -----BEGIN PRIVATE KEY-----
      ...
      -----END PRIVATE KEY-----
    certtype: "user"
    state: present
```

### CODE REVIEW

File: library/pfsense_cert.py [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+---
+module: pfsense_cert
+version_added: "0.1"
```
Comment (opoplawski) [inline library/pfsense_cert.py:17]: Let's make this 0.4.3.

**Comment by opoplawski** [comment]
Let's collapse into a single commit as well.  Thanks!

**Comment by opoplawski** [comment]
This should fix #46

**Comment by opoplawski** [comment]
Thank you for the contribution.  Can you fixes the issues found here: https://github.com/opoplawski/ansible-pfsense/actions/runs/1835721852

**Comment by cmarodrigues** [comment]
> Thank you for the contribution. Can you fixes the issues found here: https://github.com/opoplawski/ansible-pfsense/actions/runs/1835721852

Done.

File: library/pfsense_cert.py (hunk 1 above)
Comment (opoplawski) [reply library/pfsense_cert.py:17]: Actually, I'm going for 0.5.0 now.

**Comment by opoplawski** [comment]
Thanks, but looks like there are still some issues to resolve.

**Comment by cmarodrigues** [comment]
That should be fix the sanity tests

**Comment by opoplawski** [comment]
Thanks again for the contribution!

**Comment by cmarodrigues** [comment]
I don't mean to click on revert. Sorry.

**Pull request merged**